src/advisors/cpp_exception.ts

```diff
@@ -53,7 +53,8 @@ class CppExceptionAdvisor implements IAdvisor {
         if (
           fileExt.includes(path.extname(m.file)) &&
           !m.content.trim().startsWith("//") &&
-          !m.content.trim().startsWith("/*")
+          !m.content.trim().startsWith("/*") &&
+          !m.content.trim().endsWith("*/")
```

Comment:
Please file a different PR for #34 if you'd like to fix 😄

Reply:
Moved it to a new PR 😄